packages/chart/src/ChartUtils.ts

```diff
@@ -968,29 +966,8 @@ class ChartUtils {
       // The default histfunc in plotly is 'count', but the data passed up from the API provides explicit x/y values and bins
       // Since it's converted to bar, just set the widths of each bar
       seriesData.width = [];
-
-      if (seriesData.marker.line !== undefined) {
-        Object.assign(seriesData.marker.line, {
-          color: theme.paper_bgcolor,
-          width: 1,
-        });
```

Comment:
Just to confirm, this is an intentional change to the styling for histograms? It's not in the default template

Reply:
Yes, @dsmmcken specified that histogram bars should not have any border